applications/machine_learning/src/modules/led_state.c

```diff
@@ -193,6 +202,26 @@ static void validate_configuration(void)
 	}
 }
 
+static void ml_result_set_signin_state(bool state)
+{
+	struct ml_result_signin_event *event = new_ml_result_signin_event();
+
+	event->module_idx = MODULE_IDX(MODULE);
+	event->state = state;
+	EVENT_SUBMIT(event);
+	ml_signed = state;
+	LOG_INF("Currently %s result event", state ? "signed in" : "signed off from");
+}
+
+static bool handle_button_event(const struct button_event *event)
+{
+	if ((event->key_id == RESULT_SIGNIN_BUTTON_ID) && event->pressed) {
+		ml_result_set_signin_state(!ml_signed);
```

Comment:
Make sure to `clear_prediction` and display default LED effect when you unsubscribe

Reply:
Ok